pkg/backend/nvme.go

```diff
@@ -103,18 +82,10 @@ func (s *Server) DeleteNVMfRemoteController(_ context.Context, in *pb.DeleteNVMf
 		log.Printf("error: %v -> %v", err, volume)
 		return nil, err
 	}
-	name := path.Base(volume.Name)
-	params := spdk.BdevNvmeDetachControllerParams{
-		Name: name,
-	}
-	var result spdk.BdevNvmeDetachControllerResult
-	err := s.rpc.Call("bdev_nvme_detach_controller", &params, &result)
-	if err != nil {
-		log.Printf("error: %v", err)
-		return nil, err
+	if s.Volumes.NvmeNumberOfPaths[in.Name] != 0 {
```

Comment:
not sure this is right... according to API, path -> ctrl... so we need to create ctrl first... no ?